core-bundle/src/Twig/Interop/ContextHelper.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Interop;
+
+use Contao\Template;
+
+/**
+ * @experimental
+ */
+final class ContextHelper
+{
+    /**
+     * Adjust Contao template data to seamlessly work as a Twig context.
+     */
+    public static function fromContaoTemplate(Template $template): array
+    {
+        $context = $template->getData();
+
+        array_walk_recursive(
+            $context,
+            static function (&$value, $key): void {
+                if (\is_callable($value)) {
```

Comment:
This needs to check for closures I think. Otherwise string values like `'strtolower'` produce unexpected results 😳

Reply:
Good catch! 

Changed in 1c9f019978e0eb34f7faabf42965259d7abe4eeb.